src/MIDebugEngine/Engine.Impl/DebuggedProcess.cs

```diff
@@ -744,6 +753,34 @@ private void AddExecutablePathCommand(IList<LaunchCommand> commands)
             commands.Add(new LaunchCommand("-file-exec-and-symbols " + exe, description, ignoreFailures: false, failureHandler: failureHandler));
         }
 
+        private void DetermineAndAddExecutablePathCommand(IList<LaunchCommand> commands)
+        {
+            // Runs a shell command to get the full path of the exe.
+            string absoluteExePath = string.Format(CultureInfo.InvariantCulture, @"shell readlink -f /proc/{0}/exe", _launchOptions.ProcessId);
```

Comment:
>" [](start = 82, length = 1)

This isn't going to work if the target OS is OSX. Might want to change the bash command to fail gracefully if it isn't Linux.